common/src/main/webapp/sass/explore.scss

```diff
@@ -654,42 +698,146 @@ $search-preview-margin: 5px;
 // ------
 // Obs badge
 // ------
-
-.ui.button.obs-clone-button,
-.ui.button.delete-button.obs-delete-button {
-  border: none;
-  background-color: transparent;
+.obs-badge {
+  width: 100%;
   margin: 0;
-  padding: 0 !important;
+  padding: 1em;
+  background-color: var(--obs-badge-background);
+  border: solid 1px var(--site-border-color);
+  border-radius: 5px;
+
+  // defined in lucuma-ui.scss. Linting doesn't capture that.
+  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
```

Comment:
I'm not sure how we can capture these variables in the linting since the sass file doesn't exist until after linking is done and the sass files are extracted from the lucuma-ui jar.